package.json

```diff
@@ -3,22 +3,40 @@
   "version": "1.5.0",
   "workspaces": [
     "packages/daemon",
-    "packages/wallet-service"
+    "packages/wallet-service",
+    "packages/common"
   ],
   "engines": {
     "node": ">=18"
   },
   "nohoist": [
     "**"
   ],
-  "repository": "git@github.com:HathorNetwork/hathor-wallet-service-sync_daemon.git",
+  "repository": "git@github.com:HathorNetwork/hathor-wallet-service.git",
   "author": "André Abadesso <andre.abadesso@gmail.com>",
   "private": true,
   "devDependencies": {
-    "dotenv": "^16.3.1",
-    "mysql2": "^3.6.1",
-    "sequelize": "^6.33.0",
-    "sequelize-cli": "^6.6.1"
+    "@typescript-eslint/eslint-plugin": "^7.4.0",
+    "@typescript-eslint/parser": "^7.4.0",
+    "dotenv": "^16.4.5",
+    "eslint": "^8.57.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.9.0",
+    "mysql2": "^3.9.3",
+    "sequelize": "^6.37.2",
+    "sequelize-cli": "^6.6.2"
   },
-  "packageManager": "yarn@4.1.0"
+  "packageManager": "yarn@4.1.0",
+  "dependencies": {
+    "@aws-sdk/client-apigatewaymanagementapi": "3.540.0",
+    "@aws-sdk/client-lambda": "3.540.0",
+    "@aws-sdk/client-sqs": "3.540.0",
+    "@hathor/wallet-lib": "0.39.0",
+    "bip32": "^4.0.0",
+    "bitcoinjs-lib": "^6.1.5",
+    "bitcoinjs-message": "^2.2.0",
+    "tiny-secp256k1": "^2.2.3",
+    "winston": "^3.13.0"
```

Comment:
Added those dependencies here so the projects can use `peerDependencies` and share the dependency